.github/workflows/docs.yml

```diff
@@ -0,0 +1,69 @@
+name: docs
+
+on:
+  pull_request:
+    paths:
+      - 'docs/**'
+
+env:
+  # An envar that signals to tests we are executing in the CI environment
+  CONTINUOUS_INTEGRATION: true
+  # allow overriding Maven command
+  MAVEN: ./mvnw
+  # maven.wagon.rto is in millis, defaults to 30m
+  MAVEN_OPTS: "-Xmx512M -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
+  MAVEN_INSTALL_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
+  MAVEN_FAST_INSTALL: "-B --strict-checksums -V --quiet -T C1 -DskipTests -Dair.check.skip-all"
+  MAVEN_TEST: "-B --strict-checksums -Dair.check.skip-all --fail-at-end"
+  RETRY: .github/bin/retry
+
+# Cancel previous PR builds.
+concurrency:
+  # Cancel all workflow runs except latest within a concurrency group. This is achieved by defining a concurrency group for the PR.
+  # Non-PR builds have singleton concurrency groups.
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.event.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  docs-checks:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java-version:
+          - 11
+          - 17
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: ${{ matrix.java-version }}
+      - name: Maven Checks
+        run: |
+          $RETRY $MAVEN install -B --strict-checksums -V -T C1 -DskipTests -P ci -am -pl ':trino-docs'
+
+  test-docs:
```

Comment:
We should skip this for non-docs-only PRs, as that gets covered by ci.yml
Currently, PRs with code changes along with docs changes have this modules run twice.

Reply:
Thus, depending on the distribution of "docs only" vs "code + docs" PRs, we may have or may have not reduced CI load overall.

Reply:
Right, we should skip docs.yml when the PR contains both code + docs change.